docs: improve wild(*) section (#1228)
bareos/bareos: docs/manuals/source/Configuration/Director.rst

   in turn until the first one that matches.  Note, if you exclude a
   directory, no files or directories below it will be matched.

   It is recommended to enclose the string in double quotes.

   You may want to test your expressions prior to running your
   backup by using the :ref:`bwild` program.
   You can also test your full FileSet definition by using
   the :ref:`estimate <estimate>` command.

   An example of excluding with the WildFile option is presented

>    below: :ref:`FileSetExamples`

Reviewer: ```suggestion
   at :ref:`FileSetExamples`
```

   You can also test your full FileSet definition by using
   the :ref:`estimate <estimate>` command.

   An example of excluding with the WildFile option is presented

>    below: :ref:`FileSetExamples`

Reviewer: ```suggestion
   at :ref:`FileSetExamples`
```

   backup by using the :ref:`bwild` program.
   You can also test your full FileSet definition by using
   the :ref:`estimate <estimate>` command.

   An example of excluding with the WildFile option is presented

>    below: :ref:`FileSetExamples`

Reviewer: ```suggestion
   at :ref:`FileSetExamples`
```

   Specifies a POSIX extended regular expression to be applied to the
   filenames and directory names, which include the full path.  If :strong:`
   Exclude` is not enabled, the regex will select which files are to be

>    included. If :strong:`Exclude=yes` is specified, the regex will select

Reviewer: ```suggestion
   included. If :config:option:`dir/fileset/include/options/Exclude = yes` is specified, the regex will select
```

The "trick" here was to add a RegexFile expression that matches all files. It does not match directory names, so all directories in /myfile will be backed up (the directory entry) and any \*.Z and \*.gz files contained in them. If you know that certain directories do not contain any \*.Z or \*.gz files and you do not want the directory entries backed up, you will need to explicitly exclude those directories. Backing up a directory entries is not very expensive.

> Bareos uses the system regex library and some of them are different on different OSes. The above has been reported not to work on FreeBSD.

Reviewer: ```suggestion
Bareos uses the system regex library and some of them are different on different OSes.
```

-  If you are using the old Exclude syntax (noted below), you may not specify a drive letter in the exclude. The new syntax noted above should work fine including driver letters.

> Thanks to Thiago Lima for summarizing the above items for us. If you are having difficulties getting includes or excludes to work, you might want to try using the :bcommand:`estimate job=job-name listing` command documented in the :ref:`Console chapter <estimate>` of this manual.

Reviewer: ```suggestion
Thanks to Thiago Lima for summarizing the above items for us. If you are having difficulties getting includes or excludes to work, you might want to try using the :bcommand:`estimate job=job-name listing` command documented in the :ref:`estimate` section of this manual.
```
